.github/scripts/generate_ci_workflows.py

```diff
@@ -469,6 +470,20 @@ def generate_workflow_file(self, workflow_template: jinja2.Template) -> None:
             labels={LABEL_CIFLOW_DEFAULT, LABEL_CIFLOW_LINUX, LABEL_CIFLOW_CPU, LABEL_CIFLOW_XLA, LABEL_CIFLOW_NOARCH},
         ),
     ),
+    CIWorkflow(
+        arch="linux",
+        build_environment="periodic-pytorch-linux-xenial-cuda10.2-cudnn7-py3-gcc7-old-gradcheck",
```

Comment:
nit: I realized after the fact that the label says "slow" while the build name says "old". It would be nice to fix that discrepancy and call everything "slow".
This might be a good time to make that change to avoid breaking things again in the future?